zipline/pipeline/filters/filter.py

```diff
@@ -533,3 +535,34 @@ class StaticAssets(StaticSids):
     def __new__(cls, assets):
         sids = frozenset(asset.sid for asset in assets)
         return super(StaticAssets, cls).__new__(cls, sids)
+
+
+class NoMissingValues(CustomFilter, SingleInputMixin):
+    """Pipeline filter indicating input term has data for a given window.
+    """
+
+    def __new__(cls,
+                inputs,
+                window_length,
+                mask=NotSpecified):
+
+        if isinstance(inputs[0], Filter):
+            raise ValueError(
+                "Input to filter `NoMissingValues` cannot be a Filter."
+            )
+
+        return super(NoMissingValues, cls).__new__(
+            cls,
+            inputs=inputs,
+            window_length=window_length,
+            mask=mask,
+        )
+
+    def compute(self, today, assets, out, value):
+        if isinstance(value, LabelArray):
+            out[:] = ~np_any(value.is_missing(), axis=0)
```

Comment:
One thing that probably **is** worth doing here, though, is passing `out=out` to np.any instead of assigning the result to `out[:]`.

Reply:
That would make a difference, however the output should be the negation of the `np.any` results.

Reply:
Would moving the negation inside work? `np_all(~value.is_missing(), axis=0, out=out)`

Reply:
That wouldn't work in the case where an asset has all filled in data.

For the case` value = [[3.], [4.], [5.]]`, the inner expression resolves to [[F], [F], [F]]. The function `np.all` would return `False` for that asset, even though it is not missing any values.

Reply:
That doesn't work in the case where some of the asset's values are missing, but `np.all` would.

Reply:
Ah, nvm, didn't see that @richafrank suggested `np.all` already.